MMM-COVID19.js

```diff
@@ -76,6 +76,10 @@ Module.register("MMM-COVID19", {
     return this.config.header
   },
 
+  numberWithCommas: function(x) {
```

Comment:
this should be places at the bottom next to `compareValues` and please put some comments before the function

Reply:
done in 803276d